definitions/apache_module.rb

```diff
@@ -17,7 +17,7 @@
 # limitations under the License.
 #
 
-define :apache_module, :enable => true, :conf => false, :restart => false do # rubocop:disable Metrics/BlockLength
```

Comment:
This will break in newer versions of rubocop, the real way to fix this is to refactor as part of #414 .